controller/comments.go

```diff
@@ -112,10 +113,16 @@ func (c *CommentsController) Delete(ctx *app.DeleteCommentsContext) error {
 		if err != nil {
 			return jsonapi.JSONErrorResponse(ctx, err)
 		}
-		if *identityID != cm.CreatedBy {
```

Comment:
thanks, @alexeykazakov I am adding that back, me too should have noticed it earlier while coding. sorry.
Will push the code soon. tests are failing :)